### PR TITLE
Add Trino Fest 2024 recap

### PR DESCRIPTION
**Pull request by mosabua**
A bunch more things to do .. will probably change date again to Monday.

### CODE REVIEW

File: _posts/2024-06-18-trino-fest-recap.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: "Trino Fest 2024 recap"
+author: "Manfred Moser, Cole Bowden, Monica Miller"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2024/recap-banner.png
+---
+
+tbd
+<!--more-->
```
Comment (monimiller) [inline _posts/2024-06-18-trino-fest-recap.md:10]: Trino Fest 2024 is successfully in the books! While over 100 enthusiastic
members of the community gathered in Boston, over 650 virtual attendees 
joined us worldwide to learn from our expert speakers as they discussed
topics such as table formats, enhancements and optimizations, and use
cases with Trino both large and small.

Comment (monimiller) [inline _posts/2024-06-18-trino-fest-recap.md:10]:
```suggestion
Trino Fest 2024 is successfully in the books! While over 100 enthusiastic
members of the community gathered in Boston, over 650 virtual attendees
joined us worldwide to learn from our expert speakers as they discussed
topics such as table formats, enhancements and optimizations, and use
cases with Trino both large and small.
<!--more-->
```

File: _posts/2024-06-18-trino-fest-recap.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+tbd
+
+
+## Sessions
+
+* [What’s new in Trino this summer](https://www.youtube.com/watch?v=){:target="_blank"}
+  presented by Martin Traverso from
+  [Starburst](https://www.starburst.io){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Reducing query cost and query runtimes of Trino powered analytics platforms](https://www.youtube.com/watch?v=){:target="_blank"}
```
Comment (monimiller) [inline _posts/2024-06-18-trino-fest-recap.md:22]:
```suggestion
* [Reducing query cost and query runtimes of Trino powered analytics platforms](https://youtu.be/mk3n0_tAdZY?si=8d1oyZmpxL2Luj_5){:target="_blank"}
```

Comment (monimiller) [inline _posts/2024-06-18-trino-fest-recap.md:22]:
```suggestion
* [Reducing query cost and query runtimes of Trino powered analytics platforms](https://youtu.be/11yhPXIXiBY?si=O4x4ac2X8Kq0QPmA){:target="_blank"}
```

File: _posts/2024-06-18-trino-fest-recap.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  presented by Jonas Irgens Kylling from
+  [Dune](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Enhancing Trino's query performance and data management with Hudi: innovations and future](https://www.youtube.com/watch?v=){:target="_blank"}
```
Comment (monimiller) [inline _posts/2024-06-18-trino-fest-recap.md:26]:
```suggestion
* [Enhancing Trino's query performance and data management with Hudi: innovations and future](https://youtu.be/JMzS2BbeK0E?si=wkJxUGt6jsG6-xz2){:target="_blank"}
```

File: _posts/2024-06-18-trino-fest-recap.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+tbd
+<!--more-->
+
+## Re
+tbd
+
+
+## Sessions
+
+* [What’s new in Trino this summer](https://www.youtube.com/watch?v=){:target="_blank"}
```
Comment (monimiller) [inline _posts/2024-06-18-trino-fest-recap.md:18]:
```suggestion
* [What’s new in Trino this summer](https://www.youtube.com/watch?v=mk3n0_tAdZY){:target="_blank"}
```

File: _posts/2024-06-18-trino-fest-recap.md [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Reducing query cost and query runtimes of Trino powered analytics platforms](https://www.youtube.com/watch?v=){:target="_blank"}
+  presented by Jonas Irgens Kylling from
+  [Dune](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Enhancing Trino's query performance and data management with Hudi: innovations and future](https://www.youtube.com/watch?v=){:target="_blank"}
+  presented by Ethan Guo from
+  [Onehouse](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Trino Engineering @ Microsoft](https://www.youtube.com/watch?v=){:target="_blank"}
```
Comment (monimiller) [inline _posts/2024-06-18-trino-fest-recap.md:30]:
```suggestion
* [Trino Engineering @ Microsoft](https://www.youtube.com/watch?v=t7ndqYUhKSA){:target="_blank"}
```

File: _posts/2024-06-18-trino-fest-recap.md [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+  presented by George Fisher and Ishan Patwa from
+  [Microsoft](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Enhancing data governance in Trino with the OpenLineage integration](https://www.youtube.com/watch?v=){:target="_blank"}
```
Comment (monimiller) [inline _posts/2024-06-18-trino-fest-recap.md:34]: [](url)
```suggestion
* [Enhancing data governance in Trino with the OpenLineage integration](https://www.youtube.com/watch?v=A7hj1M7IYj8){:target="_blank"}
```

File: _posts/2024-06-18-trino-fest-recap.md [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+  presented by Alok Kumar Prusty from
+  [Apple](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Best practices and insights when migrating to Apache Iceberg for data engineers](https://www.youtube.com/watch?v=){:target="_blank"}
```
Comment (monimiller) [inline _posts/2024-06-18-trino-fest-recap.md:38]:
```suggestion
* [Best practices and insights when migrating to Apache Iceberg for data engineers](https://www.youtube.com/watch?v=dKQ2zShNlyQ){:target="_blank"}
```

File: _posts/2024-06-18-trino-fest-recap.md [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+  presented by Amit Gilad from
+  [Cloudinary](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Trino query intelligence: insights, recommendations, and predictions](https://www.youtube.com/watch?v=){:target="_blank"}
```
Comment (monimiller) [inline _posts/2024-06-18-trino-fest-recap.md:42]:
```suggestion
* [Trino query intelligence: insights, recommendations, and predictions](https://www.youtube.com/watch?v=K3iSXOJNaSQ){:target="_blank"}
```

File: _posts/2024-06-18-trino-fest-recap.md [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+  presented by Marton Bod from
+  [Apple](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [The open source journey of the Trino Delta Lake Connector](https://www.youtube.com/watch?v=){:target="_blank"}
```
Comment (monimiller) [inline _posts/2024-06-18-trino-fest-recap.md:46]:
```suggestion
* [The open source journey of the Trino Delta Lake Connector](https://www.youtube.com/watch?v=mPfRYdvDcMo){:target="_blank"}
```

File: _posts/2024-06-18-trino-fest-recap.md [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+  presented by Marius Grama from
+  [Starburst](https://www.starburst.io){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Tiny Trino; new perspectives in small data](https://www.youtube.com/watch?v=){:target="_blank"}
```
Comment (monimiller) [inline _posts/2024-06-18-trino-fest-recap.md:50]:
```suggestion
* [Tiny Trino; new perspectives in small data](https://www.youtube.com/watch?v=ZcY9LJDdB6Y){:target="_blank"}
```

File: _posts/2024-06-18-trino-fest-recap.md [hunk 11]
```diff
@@ -0,0 +1,116 @@
...
+  presented by Ben Jeter and Thomas Zugibe from
+  [Executive Homes](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Bridging the divide: running Trino SQL on a vector data lake powered by Lance](https://www.youtube.com/watch?v=){:target="_blank"}
```
Comment (monimiller) [inline _posts/2024-06-18-trino-fest-recap.md:54]:
```suggestion
* [Bridging the divide: running Trino SQL on a vector data lake powered by Lance](https://www.youtube.com/watch?v=jmOsVbGfon0){:target="_blank"}
```

File: _posts/2024-06-18-trino-fest-recap.md [hunk 12]
```diff
@@ -0,0 +1,116 @@
...
+  presented by Lei Xu from LanceDB and Noah Shpak from Character.ai
+  [](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [How FourKites runs a scalable and cost-effective log analytics solution to handle petabytes of logs](https://www.youtube.com/watch?v=){:target="_blank"}
```
Comment (monimiller) [inline _posts/2024-06-18-trino-fest-recap.md:58]:
```suggestion
* [How FourKites runs a scalable and cost-effective log analytics solution to handle petabytes of logs](https://www.youtube.com/watch?v=xdCZBQJt-0g){:target="_blank"}
```

File: _posts/2024-06-18-trino-fest-recap.md [hunk 13]
```diff
@@ -0,0 +1,116 @@
...
+  presented by Arpit Garg from
+  [FourKites](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Observing Trino](https://www.youtube.com/watch?v=){:target="_blank"}
```
Comment (monimiller) [inline _posts/2024-06-18-trino-fest-recap.md:62]:
```suggestion
* [Observing Trino](https://www.youtube.com/watch?v=v7p72Ggcc5I){:target="_blank"}
```

File: _posts/2024-06-18-trino-fest-recap.md [hunk 14]
```diff
@@ -0,0 +1,116 @@
...
+  presented by Matt Stephenson from
+  [Starburst](https://www.starburst.io){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Accelerate Performance at Scale: Best Practices for Trino with Amazon S3](https://www.youtube.com/watch?v=){:target="_blank"}
```
Comment (monimiller) [inline _posts/2024-06-18-trino-fest-recap.md:66]:
```suggestion
* [Accelerate Performance at Scale: Best Practices for Trino with Amazon S3](https://www.youtube.com/watch?v=cjUUcHlUKxQ){:target="_blank"}
```

File: _posts/2024-06-18-trino-fest-recap.md [hunk 15]
```diff
@@ -0,0 +1,121 @@
...
+cases with Trino both large and small.
+<!--more-->
+
+## Re
+tbd
+
+
+## Sessions
+
+* [What’s new in Trino this summer](https://www.youtube.com/watch?v=mk3n0_tAdZY){:target="_blank"}
```
Comment (colebow) [inline _posts/2024-06-18-trino-fest-recap.md:23]: Martin recapped everything that's happened in Trino over the last six months, taking a look at the biggest new features and how Trino development is going better than ever. He also gave a sneak peek at what we can expect soon in Trino.

File: _posts/2024-06-18-trino-fest-recap.md [hunk 16]
```diff
@@ -0,0 +1,121 @@
...
+  presented by Martin Traverso from
+  [Starburst](https://www.starburst.io){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Reducing query cost and query runtimes of Trino powered analytics platforms](https://www.youtube.com/watch?v=11yhPXIXiBY){:target="_blank"}
```
Comment (colebow) [inline _posts/2024-06-18-trino-fest-recap.md:27]: Jonas gave a detailed talk about how Dune has improved their performance of Trino with a few key tweaks. That includes leveraging caching with Alluxio, advanced cluster management, and storing, sampling, and filtering query results.

File: _posts/2024-06-18-trino-fest-recap.md [hunk 17]
```diff
@@ -0,0 +1,121 @@
...
+  presented by Jonas Irgens Kylling from
+  [Dune](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Enhancing Trino's query performance and data management with Hudi: innovations and future](https://www.youtube.com/watch?v=JMzS2BbeK0E){:target="_blank"}
```
Comment (colebow) [inline _posts/2024-06-18-trino-fest-recap.md:31]: Ethan gave a look into development on Hudi and Trino's Hudi connector, explaining multi-modal indexing and how it can improve query performance. He also gave an overview of the roadmap and future of the connector.

File: _posts/2024-06-18-trino-fest-recap.md [hunk 18]
```diff
@@ -0,0 +1,121 @@
...
+  presented by Ethan Guo from
+  [Onehouse](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Trino Engineering @ Microsoft](https://www.youtube.com/watch?v=t7ndqYUhKSA){:target="_blank"}
```
Comment (colebow) [inline _posts/2024-06-18-trino-fest-recap.md:35]: George and Ishan gave a deep dive into what's been going on with Microsoft's deployment and management of Trino. This included clients and integrations, result caching, a sharded SQL connector, deep debugging and monitoring, and seamless security integration with Azure.

File: _posts/2024-06-18-trino-fest-recap.md [hunk 19]
```diff
@@ -0,0 +1,121 @@
...
+  presented by George Fisher and Ishan Patwa from
+  [Microsoft](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Enhancing data governance in Trino with the OpenLineage integration](https://www.youtube.com/watch?v=A7hj1M7IYj8){:target="_blank"}
```
Comment (colebow) [inline _posts/2024-06-18-trino-fest-recap.md:39]: Alok's lightning talk was all about how Apple deployed OpenLineage, an open framework for data lineage collection and analysis, and built a Trino plugin to publish OpenLineage complaint events that can be viewed and monitored.

File: _posts/2024-06-18-trino-fest-recap.md [hunk 20]
```diff
@@ -0,0 +1,121 @@
...
+  presented by Alok Kumar Prusty from
+  [Apple](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Best practices and insights when migrating to Apache Iceberg for data engineers](https://www.youtube.com/watch?v=dKQ2zShNlyQ){:target="_blank"}
```
Comment (colebow) [inline _posts/2024-06-18-trino-fest-recap.md:43]: Amit shared how Cloudinary expanded their data lake to use Apache Iceberg. He demonstrated how moving from Snowflake to an open table format allowed them to reduce storage costs and leverage different query and processing engines to run more powerful analytics at scale.

File: _posts/2024-06-18-trino-fest-recap.md [hunk 21]
```diff
@@ -0,0 +1,121 @@
...
+  presented by Amit Gilad from
+  [Cloudinary](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Trino query intelligence: insights, recommendations, and predictions](https://www.youtube.com/watch?v=K3iSXOJNaSQ){:target="_blank"}
```
Comment (colebow) [inline _posts/2024-06-18-trino-fest-recap.md:47]: Marton's lightning talk explored how Apple has monitored and stored metadata for every Trino query execution, then used that data for for real-time cluster dashboarding, self-service troubleshooting, and automatic generation of recommendations for users.

File: _posts/2024-06-18-trino-fest-recap.md [hunk 22]
```diff
@@ -0,0 +1,121 @@
...
+  presented by Marton Bod from
+  [Apple](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [The open source journey of the Trino Delta Lake Connector](https://www.youtube.com/watch?v=mPfRYdvDcMo){:target="_blank"}
```
Comment (colebow) [inline _posts/2024-06-18-trino-fest-recap.md:51]: Marius went into a deep dive on all the work and collaboration that's gone into making the Delta Lake connector in Trino a robust, first-class connector. Casual discussions, engineers working together, GitHub issues filed by the community, and innovative contributions have all come together, and Marius' talk shows why an open source community is so powerful.

File: _posts/2024-06-18-trino-fest-recap.md [hunk 23]
```diff
@@ -0,0 +1,121 @@
...
+  presented by Marius Grama from
+  [Starburst](https://www.starburst.io){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Tiny Trino; new perspectives in small data](https://www.youtube.com/watch?v=ZcY9LJDdB6Y){:target="_blank"}
```
Comment (colebow) [inline _posts/2024-06-18-trino-fest-recap.md:55]: Ben and Tommy explore how Executive Homes uses Trino's robust suite of integrations to handle data at a small scale. Instead of petabytes, how about a handful of gigabytes in several different systems? It's something that Trino is well-equipped to handle thanks to how well-supported it is in the data ecosystem, and they explain why.

File: _posts/2024-06-18-trino-fest-recap.md [hunk 24]
```diff
@@ -0,0 +1,121 @@
...
+  presented by Ben Jeter and Thomas Zugibe from
+  [Executive Homes](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Bridging the divide: running Trino SQL on a vector data lake powered by Lance](https://www.youtube.com/watch?v=jmOsVbGfon0){:target="_blank"}
```
Comment (colebow) [inline _posts/2024-06-18-trino-fest-recap.md:59]: Lei and Noah give an overview of LanceDB, how it works, and what makes it a great database for multimodal AI. Then they dive into a Trino connector for Lance, and explore how Trino slots into Character.AI's workload to blend analytics with training and generating new models.

File: _posts/2024-06-18-trino-fest-recap.md [hunk 25]
```diff
@@ -0,0 +1,121 @@
...
+  presented by Lei Xu from LanceDB and Noah Shpak from Character.ai
+  [](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [How FourKites runs a scalable and cost-effective log analytics solution to handle petabytes of logs](https://www.youtube.com/watch?v=xdCZBQJt-0g){:target="_blank"}
```
Comment (colebow) [inline _posts/2024-06-18-trino-fest-recap.md:63]: With nearly a petabyte of logs being managed at FourKites, it shouldn't be a huge surprise that they've turned to Trino to handle understanding and analyzing them. Arpit discusses how they've scaled log ingestion, strategically used S3 with Parquet to minimize storage costs, transformed and extracted those logs at scale, and leveraged Trino to search and explore the datasets with Superset as a frontend for visualization.

File: _posts/2024-06-18-trino-fest-recap.md [hunk 26]
```diff
@@ -0,0 +1,121 @@
...
+  presented by Arpit Garg from
+  [FourKites](){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Observing Trino](https://www.youtube.com/watch?v=v7p72Ggcc5I){:target="_blank"}
```
Comment (colebow) [inline _posts/2024-06-18-trino-fest-recap.md:67]: Starburst has built a comprehensive observability platform around Trino to better serve its users and customers. Matt explored all the components of it, including how to integrate with Jaeger, Prometheus, and ELK.

File: _posts/2024-06-18-trino-fest-recap.md [hunk 27]
```diff
@@ -0,0 +1,121 @@
...
+  presented by Matt Stephenson from
+  [Starburst](https://www.starburst.io){:target="_blank"}.
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/.pdf)
+* [Accelerate Performance at Scale: Best Practices for Trino with Amazon S3](https://www.youtube.com/watch?v=cjUUcHlUKxQ){:target="_blank"}
```
Comment (colebow) [inline _posts/2024-06-18-trino-fest-recap.md:71]: Dai's talk explores best practices to get the most out of using Trino in conjunction with Amazon S3. He discusses partitioning, scaling workloads, reducing latency, and resolving common bottlenecks, providing valuable insights for anyone trying to manage and deploy Trino with S3.

File: index.md [hunk 28]
```diff
@@ -26,10 +26,10 @@ title: Distributed SQL query engine for big data
     <div class="col-1"></div>
     <div class="col-6" style="text-align: center;">
       <p>
-        <h1 style="color: #FFFFFF;">Trino Fest</h1>
-        <h3 style="color: #FFFFFF; padding-bottom: 1rem;">13 June, 2024</h3>
+        <h1 style="color: #FFFFFF;">Trino Fest 2024</h1>
         <div style="padding-bottom: 1rem">
-          <a class="btn btn-orange" style="display: inline-grid;" href="https://www.starburst.io/info/trino-fest-2024/?utm_medium=trino&utm_source=website&utm_campaign=Global-FY25-Q2-EV-Trino-Fest-2024&utm_content=banner" target="_blank">Register now</a>
+          <a class="btn btn-orange" style="display: inline-grid;"
+          href="{% post_url 2024-06-18-trino-fest-recap %}" target="_blank">Read the recap, look at the slides, and watch video recordings</a>
```
[the remark this answers is not in the document]
Comment (mosabua) [reply index.md]: done

File: _posts/2024-06-21-trino-fest-recap.md [hunk 29]
```diff
@@ -0,0 +1,249 @@
...
+Trino Fest 2024 is successfully in the books! While over 100 enthusiastic
+members of the community gathered in Boston, over 650 virtual attendees joined
+us worldwide to learn from our expert speakers as they discussed topics such as
+table formats, enhancements and optimizations, and use cases with Trino both
+large and small. And now it is your chance to revisit the presentations or catch
+up on everything you missed.
+
+<!--more-->
+
+To write about
```
Comment (mosabua) [inline _posts/2024-06-21-trino-fest-recap.md:18]: fyi @monimiller and @colebow .. also thoughts @dain @martint ?

File: _posts/2024-06-21-trino-fest-recap.md [hunk 30]
```diff
@@ -0,0 +1,249 @@
...
+discussions, engineers working together, GitHub issues filed by the community,
+and innovative contributions have all come together, and Marius' talk shows why
+an open source community is so powerful.
+<br><i class="fab fa-youtube" style="color: red;"></i> [Video recording](https://www.youtube.com/watch?v=mPfRYdvDcMo){:target="_blank"}
+  [(Slides)]({{site.url}}/assets/blog/trino-fest-2024/delta-lake.pdf)
+
+---
+
+**Tiny Trino; new perspectives in small data**
+<br>Presented by Ben Jeter and Thomas Zugibe from [Executive Homes](){:target="_blank"}.
```
Comment (mosabua) [inline _posts/2024-06-21-trino-fest-recap.md:170]: Anyone know the URL for their website @monimiller ?

File: index.md [hunk 31]
```diff
@@ -26,10 +26,10 @@ title: Distributed SQL query engine for big data
     <div class="col-1"></div>
     <div class="col-6" style="text-align: center;">
       <p>
-        <h1 style="color: #FFFFFF;">Trino Fest</h1>
-        <h3 style="color: #FFFFFF; padding-bottom: 1rem;">13 June, 2024</h3>
```
Comment (mosabua) [inline index.md:30]: do we want a subtitle and a shorter wording on the button instead @monimiller @colebow ?

File: _posts/2024-06-24-trino-fest-recap.md [hunk 32]
```diff
@@ -0,0 +1,273 @@
...
+* *Was great to finally meet you and we had a great time at Trino Fest!*
+* *Thanks for a great event last week. It was a pleasure to meet you all.*
+
+Many of us also [met Commander Bun Bun](https://www.linkedin.com/posts/k-shreya-s_trinofest2024-bigdata-analytics-activity-7209236269774585857-p8-e?utm_source=share&utm_medium=member_desktop),
+and [we sent greetings to the remote audience as
+well](https://www.youtube.com/watch?v=4jPYpU9Jrrw).
+
+<img src="{{site.url}}/assets/blog/trino-fest-2024/cbb-manfred.jpg">
+
+The keynote, the session, and all the talk in the hallways confirmed that Trino
```
Comment (colebow) [inline _posts/2024-06-24-trino-fest-recap.md:41]:
```suggestion
The keynote, the sessions, and all the talk in the hallways confirmed that Trino
```

**Review by colebow — APPROVED** [review]
single nit
